Tif-tifs/homework_2/99-bottles-of-beer/app.js

```diff
@@ -1,12 +1,17 @@
 /*
-  Write a script that prints the lyrics to "99 Bottles of Beer on the Wall" in the terminal. 
+  Write a script that prints the lyrics to "99 Bottles of Beer on the Wall" in the terminal.
   If you're unfamiliar with the song, you can [find the lyrics here](http://www.99-bottles-of-beer.net/lyrics.html).
-  
+
   Be sure that all of your output includes proper spacing (no words or values running into each other).
 
   Hint: You can make your output appear on multiple lines by using the string \n to insert a line break.
-  
+
   For an extra challenge, improve this code to print "1 bottle" rather than "1 bottles." (Hint: read up on if/else
-  conditional statements at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else). 
+  conditional statements at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else).
 */
 
+  for (i = 99; i > 0; i--); {
```

Comment:
Logically this loop makes sense, with the exception of the `;` on this line. A semicolon says, "stop here!" so the body of the loop won't run. Removing that makes those bottles of beer just fly by. 😛